Remove references to when old features became available (#1360)
rancher/rancher-docs: docs/how-to-guides/new-user-guides/kubernetes-clusters-in-rancher-setup/register-existing-clusters.md

## Authorized Cluster Endpoint Support for RKE2 and K3s Clusters

> Authorized Cluster Endpoint (ACE) support has been added for registered RKE2 and K3s clusters. This support includes manual steps you will perform on the downstream cluster to enable the ACE. For additional information on the authorized cluster endpoint, click [here](../manage-clusters/access-clusters/authorized-cluster-endpoint.md).

Reviewer: ```suggestion
Rancher supports Authorized Cluster Endpoints (ACE) for registered RKE2 and K3s clusters. This support includes manual steps you will perform on the downstream cluster to enable ACE. For additional information on the authorized cluster endpoint, click [here](../manage-clusters/access-clusters/authorized-cluster-endpoint.md).
```

Author: Suggestion applied to applicable versions in c2585bf9c2b445a67a1649c6cae4d1cf41b890c7.
